lib/client/BootstrappedDataService.js

```diff
@@ -1,71 +1,48 @@
 "use strict";
 
-var $ = require("../application/jquery");
-var calculateSyncRequestUrl = require("../util/calculateSyncRequestUrl");
+var bootstrappedData;
+var datasAvailable = 0;
```

Comment:
Now you're just messing with me! Maybe just `dataPoints` or `bootstrappedFields`?